caffe2/CMakeLists.txt

```diff
@@ -1526,6 +1526,35 @@ if(MSVC AND BUILD_SHARED_LIBS)
   target_compile_options(torch_cpu PRIVATE "-DONNX_BUILD_MAIN_LIB")
 endif()
 
+if(MSVC)
```

Comment:
This should be guarded behind `USE_SLEEF`

Reply:
Also, do you mind moving this to (and just remove MSVC part) https://github.com/pytorch/pytorch/blob/5410385c427cf2969da4d08b8ea9e4ed4c6b82d5/aten/src/ATen/CMakeLists.txt#L423

Reply:
> This should be guarded behind `USE_SLEEF`

1. USE_SLEEF is CPP macro: https://github.com/pytorch/pytorch/blob/8c2fde1fcf17a1108557740d581e1808cc5b7d38/aten/src/ATen/cpu/vec/vec256/vec256_float_neon.h#L38-L42
2. CMAKE not contains USE_SLEEF switch. Sleef is hard dependency of torch_cpu, if vec enabled. The only option is whether use system sleef: https://github.com/pytorch/pytorch/blob/8c2fde1fcf17a1108557740d581e1808cc5b7d38/CMakeLists.txt#L424

Reply:
<img width="1353" alt="Image" src="https://github.com/pytorch/pytorch/assets/8433590/f8a1056b-339c-408c-ba17-aa1dd59bc65b">
I referenced to the "ONNX", which is linked to torch_cpu success. It was processed as submodule in caffee2/CMakeLists.txt.
So I think I can only process sleef here.